exercises/palindrome-products/palindrome_products_test.go

```diff
@@ -37,10 +38,14 @@ var testData = []struct {
 		""},
 	{4, 10, Product{}, Product{}, "No palindromes"},
 	{10, 4, Product{}, Product{}, "fmin > fmax"},
-	/* bonus curiosities.  (can a negative number be a palindrome?
-	// most say no
-	{-99, -10, Product{}, Product{}, "Negative limits"},
-	// but you can still get non-negative products from negative factors.
+
+	/* Bonus curiosities.  Can a negative number be a palindrome? Most say no
+	{-99, -10,
+		Product{},
+		Product{},
+	"Negative limits"},
```

Comment:
I wonder if we can put the bonuses in a separate slice, and then have a line that we comment out that would append the bonuses to the testData.

Does that make sense?